github.sgnl.yaml

```diff
@@ -726,24 +721,16 @@ entities:
       - name: isDraft
         externalId: isDraft
         type: Bool
-      - name: isInMergeQueue
-        externalId: isInMergeQueue
-        type: Bool
-      - name: isMergeQueueEnabled
-        externalId: isMergeQueueEnabled
-        type: Bool
```

Comment:
Why are these removed?

Reply:
These attributes were listed in the documentation, but they failed during testing. Looks like they've removed support for them.

Reply:
Are they not present in both Cloud and On Prem versions?

Reply:
They are not present in Enterprise Server which is the instance I tested on. It could be present on Enterprise Cloud, but I won't know for sure without creating a new trial because my last one expired. BTW - I have seen attributes that are present in only one of the two instances in the past, but also attributes that are listed in the documentation but not supported by other deployment type. Do you want me to change/test anything here?

Reply:
If this is tested for GH Enterprise Server, then let's rename the file accordingly before we merge. In case GH Cloud has more attributes, we'll create another catalog for it, with the extra attributes.